etc/events.xml

```diff
@@ -16,4 +16,7 @@
     <event name="taxjar_salestax_import_rates">
         <observer name="clearBoltCache" instance="Bolt\Boltpay\Observer\ClearBoltShippingTaxCacheObserver" />
     </event>
+    <event name="layout_generate_blocks_after">
```

Comment:
This observer should be declared in etc/frontend/events.xml so it doesn't unnecessarily execute in admin.